data/brands/shop/sports.json

```diff
@@ -472,6 +472,42 @@
         "shop": "sports"
       }
     },
+    {
+      "displayName": "Rapha",
+      "locationSet": {
+        "include": [
+          "au",
+          "de",
+          "dk",
+          "es",
+          "gb",
+          "hk",
+          "jp",
+          "kr",
+          "nl",
+          "tw",
+          "us"
+        ]
+      },
+      "matchTags": [
+        "shop/bicycle",
+        "shop/clothes"
+      ],
+      "tags": {
+        "brand": "Rapha",
+        "brand:en": "Rapha",
+        "brand:ja": "ラファ",
+        "brand:wikidata": "Q7293931",
+        "brand:wikipedia": "en:Rapha (sportswear)",
+        "cuisine": "coffee_shop",
```

Comment:
Add a cuisine in a sport clothes shop maybe a little strange?

My personal opinion is separate them as **IKEA** and **IKEA restaurant**.

IKEA:
https://github.com/osmlab/name-suggestion-index/blob/1854a16c88c758434bb969ddbb037b178865be62/data/brands/shop/furniture.json#L601-L612

IKEA cafe:
https://github.com/osmlab/name-suggestion-index/blob/ed5ecbc1db752eb174b0e72d3ef44c287c62c54d/data/brands/amenity/cafe.json#L1330-L1345

Reply:
Yeah I agree with @LaoshuBaby - these should really be separate entries. Another good example is how drug stores (at least in the States) are mapped as a `shop=chemist` building with a `amenity=pharmacy` node at the pharmacy counter. There's even a wiki page about this: [One feature, one OSM element](https://wiki.openstreetmap.org/wiki/One_feature,_one_OSM_element)

Reply:
Understood. The shop cafes don't warrant their own entry in the NSI as they are a small part of the shop but I will remove the food and drink tags.